README.md

```diff
@@ -614,6 +614,29 @@ other tools to extract to path information.
     $ kerl path 19.2.3
     /aux/erlangs/19.2.3
 
+### install-docsh
+
+    kerl install-docsh
+
+Install `erl` shell documentation access
+extension - [docsh](https://github.com/erszcz/docsh).
+This extends the shell with new helpers, which enable access to full
+function help (via `h/{1,2,3}`), function specs (`s/{1,2,3}`) and type
+information (`t/{1,2,3}`).
+
+Activating a docsh-enabled Erlang installation will try to create
+a `$HOME/.erlang` symlink.
+If this file exists (i.e. you have created it manually),
+please consider removing it - otherwise, docsh won't work.
+Deactivating the kerl Erlang installation will remove the symlink.
+
+Alternatively, if the file exists and you have to keep it you can extend
+it with the content of [a docsh-specific `.erlang`][docsh-dot-erlang] - this
+task is left as an exercise for the reader - and export
+`KERL_DOCSH_DOT_ERLANG=exists` to silence unwanted warnings.
+
```

Comment:
I would add here:

* If you already have an OTP installation, you will need to remove it and re-install it **before** you execute docsh, since docsh needs some environment variables of its own to be set up and managed by the activate script.

* I might add a link to a gist or some other docs which explain how to activate docsh if you already have your own `.erlang` file. (I.e., how can I manage this the hard way without automation hiding the details.)